base_pom.xml

```diff
@@ -0,0 +1,498 @@
+    <version.plugin.deploy>3.1.1</version.plugin.deploy>
+    <version.plugin.enforcer>3.0.0-M3</version.plugin.enforcer>
+    <version.plugin.exec>3.1.0</version.plugin.exec>
+    <version.plugin.failsafe>3.0.0</version.plugin.failsafe>
+    <version.plugin.fmt>2.19</version.plugin.fmt>
+    <version.plugin.gpg>3.0.1</version.plugin.gpg>
+    <version.plugin.install>3.1.1</version.plugin.install>
+    <version.plugin.jacoco>0.8.8</version.plugin.jacoco>
+    <version.plugin.japicmp>0.17.2</version.plugin.japicmp>
+    <version.plugin.jar>3.3.0</version.plugin.jar>
+    <version.plugin.javadoc>3.5.0</version.plugin.javadoc>
+    <version.plugin.projectinforeports>3.4.2</version.plugin.projectinforeports>
+    <version.plugin.shade>3.4.1</version.plugin.shade>
+    <version.plugin.sortpom>3.0.1</version.plugin.sortpom>
+    <version.plugin.source>3.2.1</version.plugin.source>
+    <version.plugin.surefire>3.0.0</version.plugin.surefire>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.amazonaws</groupId>
+        <artifactId>aws-java-sdk-bom</artifactId>
+        <version>${awssdk.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson</groupId>
+        <artifactId>jackson-bom</artifactId>
+        <version>${jackson.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>io.grpc</groupId>
+        <artifactId>grpc-bom</artifactId>
+        <version>${grpc.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-bom</artifactId>
+        <version>${netty.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>classworlds</groupId>
+        <artifactId>classworlds</artifactId>
+        <version>1.1</version>
+      </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>proto-google-common-protos</artifactId>
+        <version>2.22.0</version>
+      </dependency>
+      <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.10.1</version>
+      </dependency>
+      <dependency>
+        <groupId>com.google.errorprone</groupId>
+        <artifactId>error_prone_annotations</artifactId>
+        <version>2.18.0</version>
+      </dependency>
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>32.1.1-jre</version>
+      </dependency>
+      <dependency>
+        <groupId>com.google.http-client</groupId>
+        <artifactId>google-http-client</artifactId>
+        <version>1.43.3</version>
```

Comment:
could we extract all version numbers to the properties?

Reply:
Updated all version numbers to the properties in parent-pom.xml